src/screencast/fps_limit.c

```diff
@@ -0,0 +1,67 @@
+#include "fps_limit.h"
+#include "logger.h"
+#include "timespec_util.h"
+#include <stdint.h>
+#include <time.h>
+#include <unistd.h>
+
+#define FPS_MEASURE_PERIOD_SEC 5.0
+
+void measure_fps(struct fps_limit_state* state, struct timespec *now);
+
+void fps_limit_measure_start(struct fps_limit_state* state, double max_fps) {
+	if (max_fps <= 0.0) 
+		return;
+
+	clock_gettime(CLOCK_MONOTONIC, &state->last_time);
+}
+
+uint64_t fps_limit_measure_end(struct fps_limit_state* state, double max_fps) {
```

Comment:
Fixed this, and other cases that I found. I used some regexp to search, and updated all places that I found, including lines that were otherwise not affected by my code. For details see https://github.com/emersion/xdg-desktop-portal-wlr/pull/74/commits/6037bc3967e9ffdf47d47473e9a505db951cd80f . I hope you don't mind; if you'd like me to revert the changes that on lines that I otherwise did not change, let me now.

Reply:
Nope, this is great. Thanks for the fixes.

Reply:
Thanks! I'll mark this as resolved, then.